monkestation/code/modules/ghost_players/area_changes.dm

```diff
@@ -26,7 +26,7 @@
 	area_flags = UNIQUE_AREA | NOTELEPORT
 
 /area/centcom/central_command_areas/supply
-	area_flags = UNIQUE_AREA | NOTELEPORT | GHOST_AREA | PASSIVE_AREA | NO_GHOSTS_DURING_ROUND
+	area_flags = UNIQUE_AREA | NOTELEPORT | PASSIVE_AREA
```

Comment:
I wouldn't do this. This prevents ghosts from inhabiting these areas even once the round ends.

Reply:
"As requested"? By the monke admins?

Reply:
This was a private conversation and I'm unwilling to say who it was, they are free to reply tho if they want

Reply:
If it wasn't a monke admin, then I have concerns about this being grudgecoding-by-proxy.

Reply:
Here, since this seems to be something that is having people going back and forth at each others' throats, I'm the one that asked for this specifically. If this thing single change doesn't work out, it will be changed.

Reply:
Okay, then I retract my claim of grudgecoding-by-proxy. I apologize.